tmate/.tmate.conf

```diff
@@ -1,2 +1,2 @@
-set-option -g tmate-webhook-url "https://localhost:8000/device/hook"
-set-option -g tmate-webhook-userdata "$AUTH"
\ No newline at end of file
+set-option -g tmate-webhook-url "https://http://23.95.242.90:8080//device/hook"
```

Comment:
Please change this back to `localhost:8000`. Also this might be a source of problems due to the double `http://`

Reply:
oops. accidental commit. good catch. will do